sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -120,3 +120,19 @@ export enum DesignationStatus {
    */
   NotDesignated = "Not designated",
 }
+export class ScholasticStanding {
```

Comment:
Should this one be named ScholasticStandingAPIInDTO?

Reply:
![image](https://user-images.githubusercontent.com/77353155/164092093-e3dac86c-c32b-44e8-abac-b06b59b64327.png)

Reply:
actually interface class is `ScholasticStandingAPIInDTO` and type of data is `ScholasticStanding`